lib/functions/configuration/main-config.sh

```diff
@@ -94,7 +94,7 @@ function do_main_configuration() {
 	USEALLCORES=yes # Use all CPU cores for compiling
 
 	[[ -z $EXIT_PATCHING_ERROR ]] && EXIT_PATCHING_ERROR="" # exit patching if failed
-	[[ -z $HOST ]] && HOST="$BOARD"                         # set hostname to the board
+	[[ -z $HOST ]] && HOST="armbian"
```

Comment:
why?

Reply:
Why not? This is an issue if you have a lot of armbian devices lying around to play with, which means if you are a tester, developer, power user. User should be changing this anyway, similar to setting a username. This will have two positive consequences - first: forcing them to deal with this if annoyed and some small contribution to brand awareness.

Optionally setting this for images only?
https://github.com/armbian/os/blob/main/userpatches/config-armbian-images.conf

Reply:
Aren't especially power users, testers and developers being annoyed having multiple devices in the network sharing the same hostname?
What about a tradeoff? `HOST="armbian-$BOARD"` ?

Reply:
You are right. Perhaps too far.

That would be one option, while second would be to change default prompt to armbian and leave host as is.

Reply:
Reverted

Reply:
I came here to complain before realizing it was already reverted -- thanks